### PR TITLE
added more comments

### PR DESCRIPTION
**Pull request by bml2238**

### CODE REVIEW

File: frontend/src/app/dashboard/events/events.component.ts [hunk 1]
```diff
@@ -26,7 +26,6 @@ export class EventsComponent implements OnInit{
 
   ngOnInit() {
     //const isMobile = navigator.userAgent; //only display one event per page on mobile
```
Comment (bml2238) [inline frontend/src/app/dashboard/events/events.component.ts:28]:
```suggestion
    // const isMobile = navigator.userAgent; //only display one event per page on mobile
```

**Review by bml2238 — COMMENTED** [review]
rerunning the tests

**Review by rcb2957 — APPROVED** [review]
It's technically not a code change.

**Review by jts7382 — APPROVED** [review]
LGTM

**Pull request merged**